transformers/modeling_auto.py

```diff
@@ -172,6 +177,7 @@ class method.
         The model class to instantiate is selected as the first pattern matching
         in the `pretrained_model_name_or_path` string (in the following order):
             - contains `distilbert`: DistilBertForMaskedLM (DistilBERT model)
+            - contains `camembert`: CamemBERTModel (CamemBERT model)
```

Comment:
This `CamemBERTModel` should be `CamembertForMaskedLM`

Reply:
Indeed, too many copypasta, thanks